analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -64,7 +62,7 @@ class Analyzer(private val config: AnalyzerConfiguration) {
         val startTime = Instant.now()
 
         val actualRepositoryConfigurationFile = repositoryConfigurationFile
-            ?: locateRepositoryConfigurationFile(absoluteProjectPath)
+            ?: File(absoluteProjectPath, ORT_CONFIG_FILENAME)
```

Comment:
Commit message: Please explain why this is the "wrong direction".

Reply:
My intention was actually to not make a statement about what is write or wrong so I removed that part of the sentence.